StyleCop.Analyzers/StyleCop.Analyzers.Test/NamingRules/SA1313UnitTests.cs

```diff
@@ -529,6 +529,42 @@ public TestClass(string text, bool flag)
             await this.VerifyCSharpFixAsync(testCode, fixedCode).ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// Verify that an invalid method override will not produce a diagnostic nor crash the analyzer.
+        /// This is a regression test for #2189
+        /// </summary>
+        /// <returns>A <see cref="Task"/> representing the asynchronous unit test.</returns>
+        [Fact]
+        public async Task InvalidMethodOverrideShouldNotProduceDiagnosticAsync()
+        {
+            var testCode = @"
```

Comment:
This would be more memory efficient as a local const variable.

Reply:
➡️ This is just test code